### ISSUE TITLE
Failed to add mined block to chain with reason Invalid spend

### ISSUE THREAD

**Issue by byLovage** [iron-fish/ironfish#676]
Dec 09 20:55:10 xer yarn[1375342]: Successfully mined block 0000000000fae54611ccc3f4c4a7a4765701feb75304106b08666fb4c374cef6 (15802) has 3 transactions
Dec 09 20:55:11 xer yarn[1375342]: Failed to add mined block to chain with reason Invalid spend

I mined the block. But I got this error and my score was not reflected in my Graffiti account. This is the third time

Graffiti:Lovage

![2021-12-10_00-28-36](https://user-images.githubusercontent.com/83519314/145478551-c1adbf82-c494-447c-9499-989f91a9c3dd.jpg)

**Comment by gjfrn** [comment]
Duplicate of #619

**Comment by JonathanZheng** [comment]
I got the same error here. Today I mined 5 blocks, but they all failed to add to the chain.

Graffiti: Buster
![1639099475(1)](https://user-images.githubusercontent.com/37205772/145502280-32074fdf-375b-430f-b731-490aba7f5799.png)

**Comment by gleason86** [comment]
I also have this issue; I am running the latest docker container

```
Version       0.1.9 @ cca3c88

Node Name     NONE

Peer Identity 8iYRK4ou4Wqhddiy9atXMTo6dUfDtyLT30LeDO/kUiU=

Peer Agent    if/cli/cca3c88

Peer Port     9033

Bootstrap     test.bn1.ironfish.network

 ```

```
Successfully mined block 0000000001d3850644c8927d2a24418a38176bb7a5c026064f46150b0e39fc26 (15802) has 3 transactions
Failed to add mined block to chain with reason Invalid spend
```

```
Successfully mined block 0000000002beac5be05c89870570127e1b321a080ef9f90be94bb11f39dd8bd4 (15866) has 2 transactions

Failed to add mined block to chain with reason Invalid spend
```

**Comment by reb0rn21** [comment]
Same error:
`Successfully mined block 000000000d6e5b89fcb1e3888a2adeef0c87bae37a31e66be7c41e3d207d3456 (15905) has 3 transactions
Failed to add mined block to chain with reason Invalid spend`

**Comment by unixworld123** [comment]
i mined 4 block ,but show error ,how to solve?
Successfully mined block 00000000038112059efd30cc1b7dedf31275b3302cf80e22fd6147adabc69bda (15967) has 7 transactions
Failed to add mined block to chain with reason Invalid spend

**Comment by 1frkmtk1** [comment]
Same problem
The problem appeared after updating to version 0.1.9.
Graffiti: frkmtk

Dec 10 08:05:20 Ubuntu-2004-focal-64-minimal yarn[791133]: Successfully mined block 0000000001062614075264a21fb02dda811c0a05e95fbd74a4eb144377944b67 (15967) has 6 transactions
Dec 10 08:05:22 Ubuntu-2004-focal-64-minimal yarn[791133]: Failed to add mined block to chain with reason Invalid spend

**Comment by OrochiOrochi** [comment]
Same problem
The problem appeared after updating to version 0.1.9.
Graffiti: Orochi

Successfully mined block 00000000042c755631dcf3b5b92ad909f94c90a66ba846179a9969d50d693584 (17184) has 5 transactions 
Failed to add mined block to chain with reason Invalid spend

![image](https://user-images.githubusercontent.com/91251550/145684136-666cd89c-1a25-43a3-9ad0-ae826155a37d.png)

**Comment by deekerno** [comment]
We put out fixes for this issue with the release of v0.1.12. If you are still encountering this issue, feel free to re-open this issue.
